source/features/sync-pr-commit-title.tsx

```diff
@@ -84,11 +83,11 @@ function disableSubmission(): void {
 	getUI().remove();
 }
 
-const listeners: Deinit[] = [];
+const subscriptions: delegate.Subscription[] = [];
 
-function init(): Deinit {
-	listeners.push(
-		...onPrMergePanelOpen(updateCommitTitle),
+function init(signal: AbortSignal): Deinit {
+	subscriptions.push(
+		onPrMergePanelOpen(updateCommitTitle, signal),
```

Comment:
This can't work because the signal is managed by the global manager but this array is local and we must be able to deinit it here.

Reply:
We have 2 quick solutions here:

- drop the changes made to `onPrMergePanelOpen` and to this file
- create a local abortcontroller so that `onPrMergePanelOpen` can keep its improvement

For the latter, probably:

```js
function init() {
	// Local controller needed because the feature can be deinited locally
	const controller = new AbortControler()
	
	subscriptions.push(
		onPrMergePanelOpen(updateCommitTitle, signal),
		() => controller.abort(),
		etc
```

Other, more involved solutions:

- _only_ use a local controller and have `deinit` call that. `init` would return `controller.abort`
- add an API that allows a feature to deinit itself on demand, like:
	```js
	function deinit() {
		features.deinit(import.meta.url)
	}
	```
	This would free us from having to deal with the local `listeners/subscriptions` arrays